shared-module/bitmaptools/__init__.c

```diff
@@ -24,6 +24,7 @@
  * THE SOFTWARE.
  */
 
+#include <string.h>
```

Comment:
Was this needed or a debug artifact no longer required? I didn't see anything called that required string.h.

Reply:
Good catch thank you. I've removed this in the latest version but will wait to make a new commit until I address the below feedback as well.